Composer/packages/client/src/components/Notifications/useSurveyNotification.ts

```diff
@@ -30,7 +30,7 @@ const buildUrl = (info: MachineInfo) => {
   return `${SURVEY_URL_BASE}?${querystring.stringify(parameters)}`;
 };
 
-const getSurveyEligibility = () => {
+export const getSurveyEligibility = () => {
```

Comment:
Same here. You should be able to test the logic it covers by testing the hook with different use cases

Reply:
Removed this.